lib/resque/scheduler.rb

```diff
@@ -208,18 +210,18 @@ def enqueue_delayed_items_for_timestamp(timestamp)
         end
       end
 
+      def enqueue(config)
+        enqueue_from_config(config)
+      rescue => e
+        Resque::Scheduler::Plugin.run_failure_hooks(failure_handler, config, e)
```

Comment:
Why are there plugins on the failure handler?  Can't we just call a method on it?